package.json

```diff
@@ -14,27 +14,28 @@
 	"license": "MIT",
 	"private": true,
 	"scripts": {
-		"prepublish": "make build",
-		"vscode:prepublish": "make build",
-		"compile": "make build",
+		"prepublish": "powershell ./build.ps1 build",
```

Comment:
Shouldn't this be `pwsh` for xplat Powershell Core?
```
mcumming@V-VEDEEP-VM ~ % powershell
zsh: command not found: powershell
mcumming@V-VEDEEP-VM ~ % pwsh
PowerShell 6.2.3
Copyright (c) Microsoft Corporation. All rights reserved.

https://aka.ms/pscore6-docs
Type 'help' to get help.

PS /Users/mcumming> 
```

Reply:
Thanks fixed 🙂